packages/components/pie-button/src/defs.ts

```diff
@@ -4,28 +4,27 @@ export const variants = ['primary', 'secondary', 'outline', 'ghost', 'inverse',
 
 export interface ButtonProps {
     /**
-     * (Optional) What size the button should be.
-     * @default "medium"
+     * What size the button should be.
```

Comment:
are changes here intentional?

Reply:
I'd recommend chatting with @xander-marjoram about this :)